projects/packages/jetpack-mu-wpcom/src/features/verbum-comments/class-verbum-comments.php

```diff
@@ -250,14 +246,14 @@ public function enqueue_assets() {
 
 		wp_enqueue_script( 'verbum-settings' );
 
-		wp_enqueue_script(
+		Assets::register_script(
```

Comment:
cc @Automattic/vertex it seems that's why we had issues with our versioning! Let's fix this [one](https://github.com/Automattic/jetpack/blob/add/sso-improvements/projects/plugins/jetpack/modules/sso.php#L343) in our SSO work.

Reply:
On it 👍